File: 1966. 숫자를 정렬하자.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main(void)
{
	
	int T;
	int N;
	int tmp;
	
	int arr[50];
	
	cin>>T;
	
	
	for(int i=1; i<=T; i++)
	{
		
		cin>>N;	
		
		for(int j=0; j<N; j++)
		{
			cin>>tmp;
			arr[j] = tmp;
		}
		
		sort(arr, arr+N);
		
		cout<<"#"<<i;
		
		for(int k=0; k<N; k++)
		{
			cout<<" ";
			cout<<arr[k];	
		}
		
		cout<<endl;
		
	}	
	
	return 0;
}

File: 1966. 숫자를 정렬하자.py

T = int(input())


for i in range(T):

	N = int(input())

	arr = [ int(x) for x in input().split() ]

	arr.sort()

	print("#", i+1, end='', sep='')

	for j in arr:
		print(' ', end='')
		print(j, end='')

	print()